lib/auth0/mixins/httpproxy.rb

```diff
@@ -21,6 +21,7 @@ module HTTPProxy
                    elsif method == :delete
                      call(:delete, url(safe_path), timeout, add_headers({params: body}))
                    elsif method == :post_file
+                     body.merge!(multipart: true)
```

Comment:
why?

Reply:
The import method was not working as it was written, which I found out when the tests were finally running (instead of being skipped).

Reply:
The import method was not working as it was written, which I found out when the tests were finally running (instead of being skipped).